userManagement/views.py

```diff
@@ -30,3 +32,69 @@ def settings(request):
         "p_form": p_form,
     }
     return render(request, 'userManagement/settings.html', context)
+
+
+def userAccount(request):
+    if request.method == 'GET':
+        if not request.user.is_superuser:
+            return render(request, "security/403.html", status=403)
+
+    if request.method == 'POST':
+        c_form = userCreationForm(request.POST)
+        if c_form.is_valid():
+            c_form.save()
+            messages.success(request, f'Account was successfully created!')
+            return redirect('user-account')
+    else:
+        c_form = userCreationForm(request.POST)
+
+    context = {
+        'staff': User.objects.all(),
+        'c_form': c_form,
+    }
+
+    return render(request, 'userManagement/userAccount.html', context)
+
+#TODO The page is to create a user account when user clicks the link in the invitation email
+def userRegister(request):
+    if request.method == 'POST':
+        c_form = userCreationForm(request.POST)
+        if c_form.is_valid():
+            c_form.save()
+            messages.success(request, f'Account was successfully created!')
+            return redirect('user-account')
+    else:
+        c_form = userCreationForm(request.POST)
+
+    context = {
+        'staff': User.objects.all(),
+        'c_form': c_form,
+    }
+
+    return render(request, 'userManagement/userAccount.html', context)
+
+
+def userDelete(request, email):
```

Comment:
require staff annotation

Reply:
Don't understand the staff annotation

Reply:
@staff_member_required, if you put that above the method,it  will require the user to have is_staff = true